bin/adhoc-scripts/createPileupObjects.py

```diff
@@ -96,7 +96,10 @@ def parseCampaigns(campDocs, logger):
         # for each secondary dataset, create one pileup document
         for puName, puRSEs in camp.get("Secondaries", {}).items():
             puDoc = getPUSchema(puName, pileupDocs, logger)
-            puType = "premix" if puName.split("/")[-1] == "PREMIX" else "classic"
+            if puName.startswith("/Neutrino") or puName.split("/")[-1] == "PREMIX":
```

Comment:
since it starts diverging from original I rather suggest to take these patterns and provide options in script to allow specify them with default values as these two. But I don't want to block it if you think we need it quickly, I only care if more pattern will appear in a long run.

Reply:
It should be the other way around, which is also confirmed from a document that Dima collected for PREMIX data placement. AFAIK, for the last year or two, all the premix pileup samples have the PREMIX datatier. 
This "Neutrino" bit is just to deal with "legacy" pileup.